src/components/TimelineScrubber/TimelineScrubber.scss

```diff
@@ -1,3 +1,13 @@
 .timeline-scrubber {
-
-}
\ No newline at end of file
+  height: 10px;
+  background: #efefef;
+  border-radius: 5px;
+  position: relative;
+  cursor: pointer;
+  border: 2px solid transparent;
+  box-sizing: content-box;
+  &:hover {
+    border: 2px solid #888;
+    transition: border-color 1.5s linear;
```

Comment:
Transition is a little abrupt when you mouse-out, maybe worth pulling this up

Reply:
removed.